packages/typespec-python/src/emitter.ts

```diff
@@ -253,9 +255,13 @@ function getType(context: SdkContext, type: EmitterType): any {
     if (isEmptyModel(type)) {
         // do not generate model for empty model, treat it as any
         newValue = { type: "any" };
-    } else {
+    } else if (type.kind === "ModelProperty") {
```

Comment:
this isn't just for model property, we also want this for parameters as well

Reply:
All the parameter type should be `ModelProperty`, refer: https://github.com/microsoft/typespec/blob/df55792422ecbebd58a12c92f357054f0b9607db/packages/http/lib/http-decorators.tsp.

Reply:
I don't understand how to judge `all the parameter type should be ModelProperty`. Could you explain it?

Reply:
> I don't understand how to judge `all the parameter type should be ModelProperty`. Could you explain it?

You could see for all the decorator target for query, body and head parameters, they are all `ModelProperty`,